bin/chat_server_log.log

```diff
@@ -0,0 +1,4192 @@
+Apr 04, 2017 5:37:42 PM codeu.chat.ServerMain main
```

Comment:
On the other hand, it's fine to store the button image, as it's small, and it makes sense to version images.